web/pdf_rendering_queue.js

```diff
@@ -33,6 +33,9 @@ class PDFRenderingQueue {
     this.pdfThumbnailViewer = null;
     this.onIdle = null;
     this.highestPriorityPage = null;
+    /**
+     * @type {ReturnType<typeof setTimeout>}
```

Comment:
Note that the code in the `web/`-folder is only really intended to run in browsers, and please make this one-line comment instead; i.e. something like `/** @type ... */`

Reply:
oh, in that case I will replace the `setTimeout` with `window.setTimeout` to disambiguate that we want the browser typings